pkg/app-launcher/pages/index.vue

```diff
@@ -337,9 +351,17 @@ export default {
       <div class="search-input">
         <input v-model="searchQuery" :placeholder="$store.getters['i18n/t']('appLauncher.filter')" />
       </div>
-      <button class="icon-button" @click="toggleSortOrder" v-if="selectedView === 'grid'">
-        <i class="icon icon-sort" />
-      </button>
+      <div class="sort-buttons" v-if="selectedView === 'grid'">
+        <div class="sort-button" :class="{ active: sortButtons.asc }" :disabled="!sortButtons.asc" @click="toggleSortOrder('asc')">
```

Comment:
This seems a bit backwards to me. With how it is, the one that is blue that you can click on is the opposite of the actual sort order. I see the reasoning, but that is just not how toggles are typically done and would be confusing to a user. We should make the blue one that is "lit up" the active sort order. Also, we can leave both of these (including the `A-Z` in the middle) as just one big toggle.